cli/cmd/encore/db.go

```diff
@@ -28,9 +29,31 @@ var (
 	resetAll bool
 	testDB   bool
 	shadowDB bool
-	nsName   string
+	role     = cmdutil.Oneof{
+		Value:   "reader",
+		Allowed: []string{"reader", "writer", "admin", "superuser"},
```

Comment:
I think read/write/admin/superuser is better than reader/writer

Reply:
But maybe `--write` etc instead of `--role write` etc?

Reply:
problem with --write is what to do with combinations. Seems cleaner to have a role?